CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 2.1.0 (Unreleased)
 
+* Add bidirectional toxics #132
```

Comment:
Could we add a little bit more of a comment here on why this is useful? What it allows us to do?